### PR TITLE
feat(lib1): created code-base with abstract class and readme

### LINKED ISSUE #2: Setup Code Base for Libary 1
## User story
1. As a team
2. I want to set up the development project
3. So that we have a good basis for software development

## Acceptance criteria
* project can run locally
* the Kotlin Android code base is initialized (if you are unsure about the technical details mail Philip and Tim first) 
* the software project builds
* add readme how to build it locally

### PR DESCRIPTION
**Pull request by a-miscellaneous**
- abstract class gives a guideline for further implementation
- readme explains setup, execution and implementation

closes #2

### CODE REVIEW

File: VideoGenerator/src/main/kotlin/AbstractVideoGenerator.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+    /**
+     * Adds multiple frames to the video.
+     * Reuses the addFrame() method.
+     *
+     *
+     * @param framePaths The paths to the frames to be added. Image type is dynamic.
+     */
+    fun addFrame(framePaths: List<String>) {
```
Comment (akriese) [inline VideoGenerator/src/main/kotlin/AbstractVideoGenerator.kt:42]:
```suggestion
    fun addFrames(framePaths: List<String>) {
```

Comment (akriese) [reply VideoGenerator/src/main/kotlin/AbstractVideoGenerator.kt:42]: Seems this would be better :)

**Review by akriese — CHANGES_REQUESTED** [review]

**Comment by a-miscellaneous** [comment]
don't merge, still awaiting ktlint

**Comment by akriese** [comment]
#11 was just merged. You can run `ktlint` now and update this :)

**Pull request merged**